app/services/authentication/authenticator.rb

```diff
@@ -0,0 +1,127 @@
+module Authentication
+  # Authenticator will perform one of these tree operations:
+  # 1. create a new user and match it to its authentication identity
```

Comment:
This is not Markdown, so the numbers in the list don't auto-increment 😉

Reply:
LOL, going to fix it :D